server/src/main/java/io/spine/server/entity/EventBlackList.java

```diff
@@ -43,7 +43,7 @@ private EventBlackList(ImmutableSet<EventClass> forbiddenEvents) {
      * Creates a new instance of {@code EventBlackList} discarding events of the given types.
      *
      * @param eventClasses
-     *         the not allowed event classes
+     *         the list of Event classes which cannot be used
```

Comment:
I don't think we need to have event capitalized here. We're not referring to `Event` as a class. We are speaking about “event” as a specific kind of messages.